paasta_tools/cli/cmds/validate.py

```diff
@@ -149,6 +165,31 @@ def get_service_path(service, soa_dir):
             return None
 
 
+def path_to_soa_dir_service(service_path):
+    soa_dir = os.path.dirname(service_path)
+    service = os.path.basename(service_path)
+
+    return soa_dir, service
+
+
+def validate_chronos(service_path):
+    soa_dir, service = path_to_soa_dir_service(service_path)
+    instance_type = 'chronos'
+
+    for cluster in list_clusters(service, soa_dir, instance_type):
+        for instance in list_all_instances_for_service(service, instance_type, soa_dir):
+            cjc = load_chronos_job_config(service, instance, cluster, False, soa_dir)
+            (checks_passed, check_msgs) = cjc.validate()
+
+            # Remove duplicate check_msgs
+            unique_check_msgs = list(OrderedDict.fromkeys(check_msgs))
```

Comment:
You would think that this wouldn't be needed. However, without it, you wind up with duplicate messages for errors with certain fields like cpu and mem. I believe this is at least partially due to these lines in chronos_tools.py.validate

```
# Use InstanceConfig to validate shared config keys like cpus and mem
error_msgs.extend(super(ChronosJobConfig, self).validate())
```

I can create a new issue to look into this, but for the time being, the de-duplication is necessary. I will switch to using `list(set(check_msgs))` though.